docs/includes/apiargs-method-watch-option.yaml

```diff
@@ -13,14 +13,53 @@ arg_name: option
 name: fullDocument
 type: string
 description: |
-  Allowed values are 'default' and 'updateLookup'. Defaults to 'default'.
-  When set to 'updateLookup', the change notification for partial updates will
-  include both a delta describing the changes to the document, as well as a
-  copy of the entire document that was changed from some time after the change
-  occurred. The following values are supported:
+  Determines how the "fullDocument" response field will be populated for update
```

Comment:
@aleksandr-rudo: I realized after submitting https://github.com/mongodb/mongo-php-library/pull/910#issuecomment-1095093276 that I forgot to add docs to this PR. Please take a look when you get a chance.

Per the commit message, I intentionally omitted mention of the "default" and "off" constants, since they're redundant.